doc/reinstall.md

```diff
@@ -0,0 +1,118 @@
+# Cluster Reinstallation
+
+You can reinstall a cluster previously installed using the Image Based Install Operator on new hardware while preserving the cluster crypto, identity, and access.
+
+## Prerequisites
+
+- The cluster must have been originally installed using the Image Based Install Operator
+- The following secrets must still be present in the hub cluster in the same namespace as the cluster deployment
+    - `<clustername>-admin-kubeconfig`
+    - `<clustername>-admin-password`
+    - `<clustername>-seed-reconfiguration`
+- The new machine must have been installed with the desired seed image and ready for final reconfiguration
+
+## Reinstallation
+
+### Backup existing secrets
+
+Execute the following commands to back up the existing secrets where `$CDNAME` is the name of the previous `ClusterDeployment`
+
+```
+oc get secret $CDNAME-admin-kubeconfig -oyaml > admin-kubeconfig.yaml
+oc get secret $CDNAME-admin-password -oyaml > admin-password.yaml
+oc get secret $CDNAME-seed-reconfiguration -oyaml > seed-reconfiguration.yaml
+```
+
+### Prepare the secrets for recreation
+
+Once the secrets are backed up some data needs to be removed to ensure they work correctly when recreated.
+
+Remove the following fields if they are present from all the previously created secret files:
+- metadata.creationTimestamp
+- metadata.labels (remove all except `image-based-installed.openshift.io/created`)
```

Comment:
Should only the `image-based-installed.openshift.io/created` label be retained, or is it possible to keep the entire `metadata.labels` object?